metricflow-semantics/extra-hatch-configuration/requirements.txt

```
@@ -1,4 +1,5 @@
-dbt-semantic-interfaces==0.6.0.dev2
+
+dbt-semantic-interfaces>=0.5.0, <2.0.0 # dbt-mantle depends on metricflow-semantics, so DSI must always point to a production version here.
```

We might consider cutting back a bit here, but in theory the pip resolver should work correctly and return 0.6.0 or whatever for the various dbt builds.

Also, just for readability, let's reference the product rather than the repo in our commentary.

```suggestion
dbt-semantic-interfaces>=0.5.0, <2.0.0 # dbt cloud depends on metricflow-semantics, so DSI must always point to a production version here.
```

---

I actually find it more readable the other way. So I'll update it to include both 🙃